Frontend/src/components/calendar/CalendarModal.jsx

```diff
@@ -337,72 +349,89 @@ function CalendarModal({ toggleModal, onMeetingScheduled, isMentor }) {
   };
 
   return (
-    <div className="modal">
-      <div className="modal-content">
-        <span className="modal-close" onClick={toggleModal}>
-          ×
-        </span>
-        <h3>Schedule Meeting:</h3>
+    <Modal open={true} onClose={toggleModal}>
+      <Box
+        sx={{
```

Comment:
Feel mix inline style with css file is not a good practice

Reply:
Resolved